apps/masterbots.ai/app/b/[id]/page.tsx

```
@@ -8,65 +8,66 @@ import type { Metadata } from 'next'
 const PAGE_SIZE = 50
 
 export default async function BotThreadsPage({
-  params
+	params,
 }: {
-  params: { id: string }
+	params: { id: string }
 }) {
-  let chatbot, threads
+	let chatbot
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)